ansible/configs/ansible-middleware-workshop/post_software.yml

```diff
@@ -9,6 +9,14 @@
     - debug:
         msg: "Post-Software tasks Started"
 
+- name: Git checkout
+  hosts: bastions
+  ansible.builtin.git:
```

Comment:
missing the tasks keyword + indention for the task

Reply:
Yep, fixed, thanks :)